sklearn/utils/fixes.py

```diff
@@ -196,3 +200,21 @@ def _take_along_axis(arr, indices, axis):
 
         fancy_index = tuple(fancy_index)
         return arr[fancy_index]
+
+
+def delayed(function, *args, **kwargs):
```

Comment:
maybe add a comment like

    remove when https://github.com/joblib/joblib/issues/1071 is fixed

?